labrad/client.py

```diff
@@ -339,7 +340,21 @@ def settings(self):
         self._refresh()
         return self._attrs
 
-    _staticAttrs = ['settings', 'send', 'send_future']
+    def to_cluster(self, context=None):
+        """Get a representation of this packet as a flattenable cluster.
+
+        This method returns a representation of this packet as a cluster of
+        (context, name, records), where records is itself a cluster of
+        (setting name, value), one for each setting record in the packet.
+        """
+        if context is None:
+            context = self._kw.get('context', self._server._ctx)
+            if context is None:
+                context = (self._server._cxn.ID, 0)
+        records = tuple((rec.name, rec.flat) for rec in self._packet)
+        return FlatPacket(context, self._server.name, records)
+
+    _staticAttrs = ['settings', 'send', 'send_future', 'to_cluster']
```

Comment:
I don't think `to_cluster` is likely to conflict with setting names, but I understand your point. Do you think a plain old function that you pass the cluster to would be better?